libr/core/cmd_anal.c

```diff
@@ -4116,21 +4116,21 @@ static bool anal_fcn_data (RCore *core, const char *input) {
 		int i;
 		bool gap = false;
 		ut64 gap_addr = UT64_MAX;
-		char *bitmap = calloc (1, fcn->size);
+		char *bitmap = calloc (1, r_anal_fcn_size (fcn));
 		if (bitmap) {
 			RAnalBlock *b;
 			RListIter *iter;
 			r_list_foreach (fcn->bbs, iter, b) {
 				int f = b->addr - fcn->addr;
```

Comment:
why would this be ever correct, fcn->addr doesnt needs to be the lowest address possible in a function